addons/knobs/src/KnobStore.js

```diff
@@ -12,7 +12,9 @@ export default class KnobStore {
     this.store[key] = value;
     this.store[key].used = true;
     this.store[key].groupId = value.groupId;
-    this.callbacks.forEach(cb => cb());
+
+    clearTimeout(this.timer);
```

Comment:
comment this? non-obvious

Reply:
Knobs can now stop rate limiting themselves because it's mostly handled here.

You can notice this has massive positive impact on the responsiveness and performance of knobs overall.

Reply:
Right. I mean it could use a comment in the code, so that a reader can understand why it's done this way.

Reply:
✅